### ISSUE TITLE
Loading Screenshots cause a 404 and could not be displayed

### ISSUE THREAD

**Issue by omid1234** [maciejmaciejewski/azure-pipelines-protractor#13]
**Error Description:**
If there is an error in the test, screenshot will be taken and stored in the defined folder. So far so good, but it could not be displayed in the report tab in azure build. In the execution in the IDE i could see screenshots and inline screenshots from the app.
After looking in the dev tools from Chrome, i could see that the image could not be found, there is a HTTP 404

@maciejmaciejewski  Could you please provide support? That would be awesome :)

![image](https://user-images.githubusercontent.com/1710455/82811047-27e4b280-9e90-11ea-9182-d32125c4df9f.png)

**Code Snippet (config from protractor.config.json):**

```
    jasmine.getEnv().addReporter(
      new HtmlReporter({
        baseDirectory: 'e2e/reports',
        screenshotsSubfolder: 'image',
        jsonsSubfolder: 'jsons',
        gatherBrowserLogs: true,
        preserveDirectory: false
      }).getJasmine2Reporter()
    );
```

**Comment by omid1234** [comment]
**Additionaly providing the destination address of the images that could noct been found:**

`https://maciejmaciejewski.gallerycdn.vsassets.io/extensions/maciejmaciejewski/azure-pipelines-protractor/1.1.1/1587387961180/dist/image\0095004e-008e-00e5-00a3-00e100e6001f.png`

**of course differs from local exec in ide:**

`file:///C:/dev/e2e/reports/image\000c00d6-00c6-00c8-00c1-002f001a0018.png` where it works perfectly fine

**Comment by maciejmaciejewski** [comment]
Hey @omid1234  and sorry for late response, 
I have noticed that the address with gallerycdn is wrongly used when images are misconfigured. Could you please change `screenshotsSubfolder` from `image` dir to `screenshots`

**Comment by PandharinathJadhav-Wellsky** [comment]
Hi @maciejmaciejewski  - Same issue for me , not able to view screenshot image on azure pipeline. so changing 
 **screenshotsSubfolder** from **images** dir to change screenshotsSubfolder from image dir to screenshotswill resolve the issue?
my configuration
baseDirectory: 'e2e_report/chrome',
screenshotsSubfolder: 'images',

**Comment by maciejmaciejewski** [comment]
@PandharinathJadhav-Wellsky 
Yes, the reporting tool is expecting screenshots to be within the screenshots dir.

**Comment by PandharinathJadhav-Wellsky** [comment]
> @PandharinathJadhav-Wellsky
> Yes, the reporting tool is expecting screenshots to be within the screenshots dir.

@maciejmaciejewski 
I changed the screenshotsSubfolder: 'screenshots',  now its failing to load the report in azure with below error on console
![image](https://user-images.githubusercontent.com/54063712/87516693-6c741980-c69b-11ea-8345-2a86932a583d.png)

**Comment by maciejmaciejewski** [comment]
So screenshots are loading but there is some fishy with them, I will add some logging capability for easier debugging.
In the meantime can you provide task logs with system.debug variable set to true?

**Comment by PandharinathJadhav-Wellsky** [comment]
> So screenshots are loading but there is some fishy with them, I will add some logging capability for easier debugging.
> In the meantime can you provide task logs with system.debug variable set to true?

No, now report its not loading it self.

**Comment by maciejmaciejewski** [comment]
What I meant is that you can see in the logs that extension is getting the list of attachments but cannot load them into the report. I will add more debugging option for easier troubleshooting.

**Comment by maciejmaciejewski** [comment]
@PandharinathJadhav-Wellsky 
I have added more logs so you can have a look at the browser.

**Comment by lynn-fisher** [comment]
Hi there, I am also having some issue with the screenshots, they don't seem to be showing in the report. My config looks like this

 jasmine.getEnv().addReporter(new HtmlReporter({
      baseDirectory: `reports/${browser.foldername}/${process.env.ELEMENT}`,
      screenshots: 'images',
      jsonsSubfolder: 'jsons',
      excludeSkippedSpecs: true,
      takeScreenShotsOnlyForFailedSpecs: false,
      preserveDirectory: false
    }).getJasmine2Reporter())

Just getting broken link and the following in the console
![image](https://user-images.githubusercontent.com/34034697/94453542-b784cf80-01a8-11eb-9195-44415bbfce99.png)


Not sure what info from console is your output, if I can provide any more info let me know thanks

**Comment by maciejmaciejewski** [comment]
@PandharinathJadhav-Wellsky 
I have learned that 
`https://maciejmaciejewski.gallerycdn.vsassets.io/extensions/maciejmaciejewski/azure-pipelines-protractor/1.1.1/1587387961180/dist/image` is used as a base path to the extension and appended to urls when you have no screenshot attachments uploaded correctly. 

Could you please look at the task logs run it with `system.debug=true` and get back to me with results.
